src/backend/controllers/userController.js

```diff
@@ -472,6 +472,17 @@ exports.setWager = async (req: Request, res: Response) => {
 				.json({ msg: 'Could not find user in db'});
 		}
 
+		// One day's time. * 1000 because it's in milliseconds
+		const aDay = 60 * 60 * 24 * 1000;
+
+		// If a user has never made a wager or its been 24hrs since last stipend give them their stipend and update wagerDate
+		if (user.wagerDate == null || (new Date - user.wagerDate) > aDay) {
+			const newBalance = user.money + 5000;
+			user.money = newBalance;
+			user.wagerDate = new Date();
+			console.log('Applied stipend');
+		}
+		
 		// Add back the balance of the original wager
 		// Theres gotta be a cleaner way to do this but idk
 		const addBack = user.money + user.wager;
```

Comment:
One cleaner way is `user.money += user.wager;`

Reply:
was gettin errors doin this, maybe it was something dumb but i'm justr gonna go ahead and merge